.github/workflows/publish-to-pypi.yml

```diff
@@ -8,8 +8,22 @@ on:
     tags:
       - '*'
 jobs:
+  check-version:
+    name: Check if the version has been updated
+    outputs:
+      has_changed: ${{ steps.check.outputs.has_changed }}
+      version: ${{ steps.versions.outputs.new_version }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: aica-technology/.github/.github/actions/check-version@v0.11.0
+        id: check
+        with:
+          package_path: ./python/pyproject.toml
```

Comment:
This doesn't work unfortunately, the action was designed for ROS packages that have the version inside a `<version>` tag in an xml file. You'll have to parse out the version yourself, like I did in modulo, I'm afraid

Reply:
oops I approved too fast